arch/cpu/nrf/doxygen-group.txt

```diff
@@ -0,0 +1,9 @@
+/**
+ * \defgroup nrf The nRF's System-on-Chips
+ * \ingroup cpu
+ */
+
+/**
+ * \defgroup nrf-platforms Nordic Semiconductor nRF-powered platforms
```

Comment:
I'd add `(using nRF MDK)` here too to avoid confusion with the existing nRF52840 platform.

Reply:
Done